src/server.rs

```diff
@@ -26,16 +31,24 @@ impl Server {
         self.set_logging_verbosity()
             .context("set logging verbosity")?;
 
-        let addr = "[::1]:50051".parse().context("parse server address")?;
         let rt = MyRuntime::default();
         let img = MyImage::default();
 
-        info!("Runtime server listening on {}", addr);
+        let sock_path = Path::new(self.config.sock_path());
+        if sock_path.exists() {
```

Comment:
Do we have to check that the path has to be absolute?

Reply:
added a check